index.js

```diff
@@ -32,9 +29,7 @@ function createEngine(engineOptions) {
   // Since we're passing an object with jsx as the key, it'll override the rest.
   engineOptions = assign({}, DEFAULT_OPTIONS, engineOptions, {jsx: jsxOptions});
 
-  // Don't install the require until the engine is created. This lets us leave
-  // the option of using harmony features up to the consumer.
-  nodeJSX.install(engineOptions.jsx);
+  require('babel/register');
```

Comment:
We should probably make use of [the options available here](https://babeljs.io/docs/usage/require/). Currently we only register for `.jsx` files (or whatever is specified in the options instead). We then clear those same things from the require cache. It's sort of nice that we're expecting a different extension since otherwise we'd be clearing way more aggressively which could have bad results.

So there are a couple options. The first is to keep our extension option and just register that. The other is to take an array of extensions (to better match babel). And the third option is to use the `only` option and make sure we only transform files in the view directory. This would be a bigger change (haven't look at if we would need a different option to specify the view directory or if we can infer it).